tools/issue-labeler/src/AzureRAGService/TriageRAG.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Search.Documents;
+using Microsoft.Extensions.Logging;
+using Azure.AI.OpenAI;
+using Azure.Search.Documents.Models;
+using OpenAI.Chat;
+using Azure.Identity;
+using System.Text.Json;
+
+namespace AzureRAGService
+{
+    public interface ITriageRAG
+    {
+        IEnumerable<(T, double)> AzureSearchQuery<T>(Uri searchEndpoint, string indexName, string semanticConfigName, string field, DefaultAzureCredential credential, string query, int count);
+        string SendMessageQna(Uri openAIEndpoint, DefaultAzureCredential credential, string modelName, string instructions, string message, BinaryData structure);
```

Comment:
```suggestion
        string SendMessageQna(Uri openAIEndpoint, DefaultAzureCredential credential, string modelName, string instructions, string message, BinaryData structure);
        
```

Whitespace is our friend!